index.bs

```diff
@@ -1432,8 +1429,131 @@ following additional fields:
 Sessions {#streaming-sessions}
 ------------------------------------
 
-TODO
+To start a streaming session, a sender may send a
+streaming-session-start-request message with the following properties:
 
+: streaming-session-id
+:: Identifies the streaming session.  Must be unique for the (sender,
+    receiver) pair.  Can be used later to modify or terminate a
+    streaming session.
+
+: stats-interval
```

Comment:
Is the receiver required to support arbitrary stats intervals?

Maybe this should be a "desired" value, and the receiver is allowed to send stats at most that often, or less often if that's not possible.

Reply:
Changed to "desired".